components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -467,9 +467,11 @@ void ConversationDriver::MaybeSeedOrClearSuggestions() {
           mojom::SuggestionGenerationStatus::HasGenerated) {
     // TODO(petemill): ask content fetcher if it knows whether current page is a
     // video.
-    suggestions_.emplace_back(
-        is_video_ ? l10n_util::GetStringUTF8(IDS_CHAT_UI_SUMMARIZE_VIDEO)
-                  : l10n_util::GetStringUTF8(IDS_CHAT_UI_SUMMARIZE_PAGE));
+    if (!has_summarized_) {
+      suggestions_.emplace_back(
+          is_video_ ? l10n_util::GetStringUTF8(IDS_CHAT_UI_SUMMARIZE_VIDEO)
+                    : l10n_util::GetStringUTF8(IDS_CHAT_UI_SUMMARIZE_PAGE));
+    }
```

Comment:
another possible solution is to infer from the length of the conversation_history vector whether it's an initial conversation and add a summary suggestion, rather than maintaining an internal state like this. cc @petemill

Reply:
Initial conversation? Not sure what that means exactly. In any case it seems ok to keep internal state about `has_summarized` if it's a requirement of the UI to only show the _summarize_ button if it hasn't been clicked yet.

Reply:
I suppose the alternative (perhaps better?) is to check if any history items have `action_type == mojom::ActionType::SUMMARIZE_PAGE` (or `..._VIDEO`)

Reply:
checking history now. updated.